lab2_number6.sql

```diff
@@ -0,0 +1 @@
+SELECT DATE_FORMAT(date, "%d.%m.%Y") FROM labor_sql.battles;
```

Comment:
It would be better to rename a column
`SELECT DATE_FORMAT(date, "%d.%m.%Y") AS DATE FROM labor_sql.battles;`